awx_collection/tests/integration/targets/tower_project_update/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Update a project without waiting
+  tower_project_update:
+    name: "AWX-Collection-tests-tower_job_template-proj-{{ test_id }}"
+    organization: Default
+    wait: False
```

Comment:
Any `False` or `True` values in the test playbooks need to be lower case to pass lint check